lib/elixir/lib/system.ex

```diff
@@ -637,6 +637,37 @@ defmodule System do
     end
   end
 
+  @doc """
+  Similar to `System.cmd/3` but raises an error on non-zero exit status.
```

Comment:
Let's start with the first paragraph being the same as cmd/3 "Executes the given `command` with `args`." and this can be the second paragraph.